Frontend/RUGraph/GraphCandlestick.cpp

```diff
@@ -40,27 +40,24 @@ GraphCandlestick::~GraphCandlestick()
 
 void GraphCandlestick::draw(gfxpp* cGfx)
 {
-	// TODO: Create new container for rectangles. wxyz. Save in SDL_Rect.
-
-	SDL_Rect bgRect;
-	bgRect.x = 0;
-	bgRect.y = 0;
-	bgRect.w = 0;
-	bgRect.h = 0;
-
 	SDL_SetRenderDrawColor(cGfx->getRenderer(), getColor().r, getColor().g, getColor().b, getColor().a);
 
-	float xRange = (float)points.size(); // points per x axis
+	float xRange = (float)points.size() / 4; // points per x axis (4 values per candlestick)
 	float yRange = y_max - y_min;
 
 	float pointXGap = ((float)parent->getWidth()) / xRange;
 	float pointYGap = ((float)parent->getHeight()) / yRange;
 
 	Point2* cPoint = NULL;
 	Point2* prevPoint = NULL;
-	for (unsigned int i = 0; i < points.size(); ++i)
+
+	// points[i]->getY() = float rawOpen,
+	// points[i+1]->getY() = float rawClose,
+	// points[i+2]->getY() = float rawHigh,
+	// points[i+3]->getY() = float rawLow
+	for (unsigned int i = 0; i < points.size(); i+4) // i+4 because each candlestick has four values
```

Comment:
Forgot the assignment operator
```suggestion
	for (unsigned int i = 0; i < points.size(); i+=4) // i+=4 because each candlestick has four values
```